CryptoTechReminderSystem.Test/TestDouble/GetLateDevelopersStub.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections.Generic;
+using CryptoTechReminderSystem.Boundary;
+using CryptoTechReminderSystem.UseCase;
+
+namespace CryptoTechReminderSystem.Test.TestDouble
+{
+    public class GetLateDevelopersStub : IGetLateDevelopers
+    {
+        private readonly List<string> _lateDevelopers;
+        
+        public GetLateDevelopersStub(List<string> lateDevelopers = null)
```

Comment:
For clarity, I would probably change the tests so the channels are stubbed in, this seems like a temporary fix as the default is set indirectly.

Reply:
We've resolved this in https://github.com/chubberlisk/crypto_tech/pull/17/commits/e938a37d80ef091e14eab57a6c80e21c868d846a.